jenkins/jobs/ci/tidb/ghpr_common_test.groovy

```diff
@@ -0,0 +1,85 @@
+// REF: https://<your-jenkins-server>/plugin/job-dsl/api-viewer/index.html
+pipelineJob('tidb_ghpr_common_test') {
+    logRotator {
+        daysToKeep(90)
+    }
+    parameters {
+        stringParam("ghprbActualCommit")
+        stringParam("ghprbPullId")
+        stringParam("ghprbTargetBranch")
+    }
+    properties {
+        // priority(0) // 0 fast than 1
+        githubProjectUrl("https://github.com/pingcap/tidb")
+        pipelineTriggers {
+            triggers {
+                ghprbTrigger {
+                    cron('H/5 * * * *')
+                    gitHubAuthId('8b25795b-a680-4dce-9904-89ef40d73159')
+
+                    triggerPhrase('.*\/(run(-integration-tests|-common-test).*)')
+                    onlyTriggerPhrase(true)
+                    skipBuildPhrase(".*skip-ci.*")
+                    buildDescTemplate('PR #$pullId: $abbrTitle\n$url')
+                    whitelist('')
+                    orgslist('pingcap')
+                    whiteListTargetBranches {
+                        ghprbBranch { branch('master') }
+                        ghprbBranch { branch('^(release-)?5\\.[0-4]\\d*(\\.\\d+)?(\\-.*)?$') }
+                        ghprbBranch { branch('^(release-)?6\\.[2-9]\\d*(\\.\\d+)?(\\-.*)?$') }
```

Comment:
dose it need to support feature branches?

Reply:
/lgtm rest

Reply:
Consistent with existing behavior